Buddies.Web/src/pages/projects/[pid].tsx

```diff
@@ -111,6 +112,19 @@ const Project: React.VFC = () => {
       .catch((err) => displayErrorNotif(err));
   };
 
+  const requestToJoin = async () => {
+    if (!(typeof projectId === 'string')) {
+      alert('error');
+      return;
+    }
+    const res = await joinRequest(projectId)
+      .catch((error) => alert(error));
```

Comment:
The `alert` calls aren't very user friendly. Perhaps just show a toast with a generic error, or do something on behalf of the user?